test/NuGet.CommandLine.Test/CommandRunner.cs

```diff
@@ -12,7 +12,7 @@ public class CommandRunner
     {
         // Item1 of the returned tuple is the exit code. Item2 is the standard output, and Item3 
         // is the error output.
-        public static Tuple<int, string, string> Run(string process, string workingDirectory, string arguments, bool waitForExit, int timeOutInMilliseconds = 60000,
+        public static Tuple<int, string, string> Run(string process, string workingDirectory, string arguments, bool waitForExit, int timeOutInMilliseconds = 600000000,
```

Comment:
1. Please while you change code, make the lines shorter (no longer than 120 chars). It makes reviewing a PR much easier. The easy way to see this is in a github PR, you never want to scroll.
2. Why is the timeout changing? Unless there is a very good reason it shouldn't because a failed test will halt the build for a long time